core/src/main/scala/akka/persistence/jdbc/state/DurableStateQueries.scala

```diff
@@ -47,7 +47,9 @@ import akka.persistence.jdbc.config.DurableStateTableConfiguration
     durableStateTable.filter(_.persistenceId === persistenceId)
 
   private[jdbc] def insertDbWithDurableState(row: DurableStateTables.DurableStateRow, seqNextValue: String) = {
-    sqlu"""INSERT INTO #${durableStateTableCfg.tableName}
+    val tableAndSchema = durableStateTableCfg.schemaName.fold(durableStateTableCfg.tableName)(schema =>
```

Comment:
can this be moved to an instance `val` of the class to avoid this for each call?

Reply:
Moved to an instance val.